CMSgov/design-system
# [WNMGDS-1276] Convert PrivacySettingsDialog to TypeScript

## Summary
https://jira.cms.gov/browse/WNMGDS-1276

### Changed
- Converted PrivacySettingsDialog and subcomponents to TypeScript
- Add a Storybook story for the Footer component so we can test the PrivacySettingsDialog

## How to test
1. `yarn build && yarn storybook:healthcare`
2. View Storybook Footer story
3. Click on the privacy settings link
4. Change values and save
5. Open the dialog again and make sure your settings were saved
